### ISSUE TITLE
Multi-scale Interactive Network for Salient Object Detection 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#33]
# TL;DR
In the salient object detection (SOD) ,they proposed MINet, which improves accuracy by integrating information with neighboring resolutions while using multiple resolutions. The results are comparable to or better than those of SOTA in previous studies in many tasks.
![image](https://user-images.githubusercontent.com/24524018/89614082-a3b2a080-d8be-11ea-8099-42bc3f877352.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.09062

# Submission Dates(yyyy/mm/dd)
2020/07/17

# Authors and institutions
Youwei Pang, Xiaoqi Zhao, Lihe Zhang, Huchuan Lu
- Dalian University of Technology, China
- Peng Cheng Laboratory 

# Methods

# Results
![image](https://user-images.githubusercontent.com/24524018/89614086-a6ad9100-d8be-11ea-9f15-d471bebf6c17.png)

# Comments
CVPR2020
